<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit_ip">
        <property name="minimumSize">
         <size>
          <width>241</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>241</width>
          <height>41</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_port">
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>41</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_usr">
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>41</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_pwd">
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>41</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_downloadpath">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>91</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>91</height>
         </size>
        </property>
        <property name="text">
         <string>下载路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_download">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>91</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>91</height>
         </size>
        </property>
        <property name="text">
         <string>下载</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_downfilepath">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_downrename">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_upload">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>91</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>91</height>
         </size>
        </property>
        <property name="text">
         <string>上传路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_start">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>91</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>91</height>
         </size>
        </property>
        <property name="text">
         <string>上传文件</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit_upfilepath">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_uprename">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_result">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>181</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>181</height>
         </size>
        </property>
        <property name="text">
         <string>结果</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_fileinfo">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_result">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
